oauthproxy_test.go

```diff
@@ -156,6 +156,32 @@ func TestEncodedSlashes(t *testing.T) {
 	}
 }
 
+func TestNewReverseProxyWithTimeOut(t *testing.T) {
+	opts := NewOptions()
+	opts.Timeout = 1 * time.Millisecond
+
+	backend := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(5 * time.Millisecond)
+	}))
+	defer backend.Close()
+
+	backendURL, _ := url.Parse(backend.URL)
```

Comment:
don't ignore errors

Reply:
This was taken almost as-is from an existing older test. I will fix this.